src/components/BankDetails/RecipientDetails/RecipientDetailsForm/RecipientDetailsForm.tsx

```diff
@@ -183,7 +183,7 @@ export default function RecipientDetailsForm({
                     className={`relative border ${
                       !!getFieldState(f.key).error
                         ? "border-red"
-                        : "border-prim"
```

Comment:
`border-prim` (border-gray-l3) could simply be mapped to `border-gray-l4` as there's no more dark variant in the new designs